datalad/distribution/get.py

```diff
@@ -273,15 +274,27 @@ def result_renderer_cmdline(res, args):
         if not isinstance(res, list):
             res = [res]
         if not len(res):
-            ui.message("Nothing was getted")
+            ui.message("Got nothing (we have it all already)")
```

Comment:
Nope. At least atm we don't know why there was nothing we actually got. Could be skipped for several reasons, not just because we already have it. We could create `success=False` - entries for everything we skip to change this, but we agreed on not to be too chatty regarding skipped files.